src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,118 @@
-package co.inventorsoft.academy.collections.model;
+package main.java.co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.DoubleStream;
+import java.util.stream.IntStream;
 
 public class Range<T> implements Set<T> {
+    private List<T> arrayList = new ArrayList<>();
+    private static List<Integer> integerList = new ArrayList<>();
+    private static List<Float> floatList = new ArrayList<>();
+    private static List<Character> characterList = new ArrayList<>();
+    private static List<Double> doubleList = new ArrayList<>();
+
+    public Range() {
+
+    }
+
+    public Range(List<T> arrayList) {
+        this.arrayList = arrayList;
+    }
+
 
     public int size() {
-        return 0;
+        return arrayList.size();
     }
 
     public boolean isEmpty() {
-        return false;
+        return arrayList.isEmpty();
     }
 
     public boolean contains(Object o) {
-        return false;
+        return arrayList.contains(o);
     }
 
     public Iterator<T> iterator() {
-        return null;
+        return arrayList.iterator();
     }
 
     public Object[] toArray() {
-        return new Object[0];
+        return arrayList.toArray();
     }
 
     public <T1> T1[] toArray(T1[] a) {
         return null;
     }
 
     public boolean add(T t) {
-        return false;
+        return arrayList.add(t);
```

Comment:
Remember, you're implementing Set interface, which means that you shouldn't have duplications